bbb-shared-notes-server/README.md

```
@@ -12,6 +12,13 @@ npm install
 Restart nginx:
 `sudo systemctl restart nginx`
 
+### Running the tests
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)